tests/test_mssqlcliclient.py

```diff
@@ -202,6 +203,37 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        db_name = 'db_' + random_str()
+        table_name = 'table_' + random_str()
+        try:
+            client = create_mssql_cli_client()
+            setup_query = u"CREATE DATABASE {0};"\
+                u"USE {0};"\
```

Comment:
>                 [](start = 0, length = 16)

The daily tests run against Azure which don't support this construct. Please run tests against azure.

Reply:
USE statement is no longer used in the unit test.